tests/resources/configuration.xml

```diff
@@ -1,5 +1,5 @@
 <configuration>
-    <dimensions mullionHeight="12" fullscreen="1" numTilesWidth="2" screenHeight="1080" mullionWidth="14" screenWidth="3840" numTilesHeight="3"/>
+    <dimensions mullionHeight="12" fullscreen="1" numTilesWidth="2" screenHeight="1080" mullionWidth="14" screenWidth="3840" numTilesHeight="3" bezelsVertical="2" bezelsHorizontal="2"/>
```

Comment:
I would really stick to bezelsPerScreenX / bezelsPerScreenY otherwise it looks like the total number of bezels.

Also, it would be better if the values in this configuration were meaningful, using bezelsPerScreenX=1, bezelsPerScreenY=0, and taking this opportunity to adjust screenWidth="3854" because if should normally include the mullionWidth to be correct